semeio/jobs/rft/gendata_rft.py

```diff
@@ -1,27 +1,86 @@
 import logging
+import pandas as pd
 
 logger = logging.getLogger(__name__)
 
 
-def _write_points(points, fname):
-    with open(fname, "w+") as fh:
-        fh.write("\n".join([str(p.get_pressure()) for p in points]))
+def _write_active_pressure_points(trajectory_df, fname):
```

Comment:
Can you split this back into three functions? Makes it slightly easier to read.

Reply:
It was two `def`s, which should have been one or three. I can change to three.

Reply:
Second  thoughts: This function is only for making the GEN_DATA information, and makes sense being grouped, the files should not be written individually.